README.md

```diff
@@ -34,6 +34,18 @@ The plugin will automatically use any of the following configuration variables,
 * `url` - The URL to your site, e.g., `https://example.com`. If none is provided, the plugin will try to use `site.github.url`.
 * `author` - Global author information (see below)
 
+If you want to enrich your feed for Feedly, create the following object with any of these optional values:
+
+```yml
+webfeeds:
+  icon: # path to a 96x96 favicon
+  logo: # path to a 30px height svg
+  cover: # path to a large 16:9 banner image
+  accentColor: # hex colour without the hash
+  related: # true = show other stories from your feed under yours
+  analytics: # Google Analytics UA-x tracking ID
```

Comment:
We're generally trying to do away with first-party support for tracking in our products, so maybe we can wait on this one.